plantuml/LinkLoss/disconnection.puml

```diff
@@ -0,0 +1,46 @@
+@startuml
+skinparam defaultTextAlignment center
+
+|App|
+
+|Link Loss Service|
+
+|Gap|
+start
+
+|Link Loss Service|
+:disconnection;
+-> event;
+
+:get_alert_level()|
+-> level;
+
+if (alert_handler != NULL ?) then (yes)
+    if (event.getReason() == CONNECTION_TIMEOUT ?) then (yes)
+        if (level != NO_ALERT ? ) then (yes)
+            if (~ in alert ?) then (yes)
+                :set alert flag to true;
+                :on_alert_requested()|
+                if (alert_timeout > 0 ms ?) then (yes)
+                    :event_queue.call_in(alert_timeout, stop_alert())|
```

Comment:
not sure we need such detail, including event queue

Reply:
Do you want me to remove details about the event queue from the PlantUML?